frontend/src/components/Header.tsx

```diff
@@ -3,27 +3,31 @@ import "../style/Header.css"
 import {useContext} from "react";
 import {AuthContext} from "../context/AuthProvider";
 
-export  default function Header() {
+export default function Header() {
     const {token} = useContext(AuthContext);
 
     return <header>
-        <div className={"row1"}>
-            <span className={"header-title"}><b>bicycle tour organizer</b></span>
-        </div>
-        {token ?
-            <div className={"row2"}>
-                <div>
-                    <NavLink to="/">Übersicht</NavLink>
-                </div>
-                <div>
-                    <NavLink to="/userpage">Useransicht</NavLink>
-                </div>
-                <div>
-                    <NavLink to="/logout">Logout</NavLink>
-                </div>
+        <div className={"header-area"}/>
+        <div className={"header-content"}>
+            <div className={"row1"}>
+
+                <span className={"header-title"}><img id="rechts" src="/bto2.png" alt="bto-logo for header" height={80}/></span>
             </div>
-            :
-            <p className={"row2-login"}>Login</p>
-        }
+            {token ?
+                <div className={"row2"}>
+                    <div>
+                        <NavLink to="/">Übersicht</NavLink>
+                    </div>
+                    <div>
+                        <NavLink to="/userpage">Useransicht</NavLink>
+                    </div>
+                    <div>
+                        <NavLink to="/logout">Logout</NavLink>
+                    </div>
+                </div>
+                :
+                <p className={"row2-login"}>Login</p>
+            }
+        </div>
```

Comment:
Könnte man evtl in eine eigene Komponente auslagern, oder?

Reply:
Ja, vom *.tsx gebe ich Dir recht, aber vom nachfolgenden CSS war es deutlich aufwendiger die Zeile in einem unabhängigen Bereich zu stylen. Daher habe ich versucht, einen Kompromiss aus funktioneller Übersichtlichkeit und Einfachheit im Styling zu erreichen